TakoBoutIt/tacoboutit-frontend
# Testing reviews

#### What's this PR do?
Updates testing to test new components for reviews, updates other testing to reflect changes.
#### How should this be manually tested?
Run npm test and make sure all tests are passing, run a coverage test, overall coverage should be ~ 75%
#### Any background context you want to provide?
Still need to add testing to app, messed with methods a little bit but was running into issues.
#### Screenshots (if appropriate)
<img width="533" alt="Screen Shot 2020-01-07 at 10 51 17 AM" src="https://user-images.githubusercontent.com/25031031/71916664-adfb9900-313b-11ea-8755-2e48dd64bed8.png">
